README.md

```diff
@@ -18,97 +20,139 @@ Important notice:
  - Atlassian does not provide support for Raspberry Pi, not matter what license you have or what OS you run on your Pi. If something is messed up, you can go to their community forums, but assume that you're on your own.
  - The primary bottleneck for running Jira on a Pi is memory. Make sure you use a lightweight OS and get rid of your other memory intensive apps.
 
-
-#0. Prerequisite
+## Prerequisite
 
 This guide was made for a Rasberry Pi 3, running a 4.9.59-v7+ RASPBIAN STRETCH WITH DESKTOP. However, Desktop was disabled using the sudo raspi-config command, based on this guide: http://ask.xmodulo.com/disable-desktop-gui-raspberry-pi.html.
 Since we run headless, with no GUI, we need SSH to access the Pi. Follow the official guide based on your configuration: https://www.raspberrypi.org/documentation/remote-access/ssh/
 
-
-#1. Increase swap
+## Increase swap
 
 The primary bottleneck is memory. If the Jira JVM can't allocate enough heap, it will crash. I know, swap is slow (but not as much since it's on an SD card), but your system is clever enough to keep hot memory in the actual RAM (or even in cache), so performance will be acceptable even with swap usage.
 Another question is how much this will kill your SD card as flash has finite write cycles, but I have no answer to that. I use 2 GB swap because why not, but I would recommend at least 1 GB.
 
- 1. Go to /etc/dphys-swapfile with your favourite editor
- 2. Edit the relevant line: CONF_SWAPSIZE=2048
+ 1. Go to `/etc/dphys-swapfile` with your favourite editor
+ 2. Edit the relevant line: `CONF_SWAPSIZE=2048`
  3. Reboot
  4. Verify e.g. with htop that your swap is now 2G. Look for the line Swp on the top.
 
+## Overclock the Raspberry
+
+If you have a case with builtin cooling (i.e. fan + heat sink) I suggest to overclock the Raspberry to get better performance. Simply edit `/boot/config.txt`.
```

Comment:
Please change "I suggest" to "rationalpsyche suggests" or similar. Take the kudos for it. (Some occurrences below as well)